scanner/config.yaml.sample

```diff
@@ -6,11 +6,17 @@ indexer:
     conn_string: "host=/var/run/postgresql"
     password_file: ""
   get_layer_timeout: 1m
+  repo:
+    rhel-repository-scanner:
+        repo2cpe_mapping_url: https://sensor.stackrox.svc/scanner/definitions?type=repo2cpe
+  package:
+    rhel_containerscanner:
+        name2repos_mapping_url: https://sensor.stackrox.svc/scanner/definitions?type=name2cpe
 matcher:
   enable: true
   database:
     conn_string: "host=/var/run/postgresql"
     password_file: ""
 mtls:
-  certs_dir: ""
-log_level: info
+  certs_dir: certs/scanner-v4
```

Comment:
It makes things simpler. Maybe we want to change the default to `certs/scanner-v4` because that's what our `Makefile` will populate when we call `make build`.